### ISSUE TITLE
Question: OmniCC & Cooldown timer

### ISSUE THREAD

**Issue by spamwax** [brittyazel/Neuron#149]
I can't seem to be able to disable the Neuron's timer while running omnicc, and as a result I get two timers on my buttons!!!
I tried toggling the related option in the bar edit mode for each bar but that didn't help.
Is this a known issue or am I doing something wrong?

**Comment by brittyazel** [comment]
Hrm, I'm not sure. I've never used Omnicc. Do you mean the text timer or the overlay clock style CD timer

**Comment by spamwax** [comment]
no, the actual text that count downs to zero (right icon on the attached screenshot). I have to use omnicc because of its integration with other mods.

To me it seems to be related to saving/persisting the settings between sessions or reloads. I have to make bag bar & cast bar hidden every time i log in, so maybe hiding the timer text also doesn't get saved and I get the duplicate.[
![world of warcraft 7_30_2018 8_06_20 pm 2](https://user-images.githubusercontent.com/1251233/43429943-637b9150-9434-11e8-91df-6b078090b47f.png)
](url)

**Comment by brittyazel** [comment]
yeah, you can disable that countdown text right from in the bar editor window. In the bottom right box there are options for cooldown text and such

**Comment by Addonman** [comment]
Its old debate where those double coodowns come from.I never used Neuron coodldows (have them turned off) and still get those on some spells especially on those with charges wher 1st charge goes normally but 2nd  gets double coldowns.
That bug is present for couple of years because Ion authors said it was Omincc problem,Omnicc said it s Ion(now Neurn ) problem and it never got fixed.
https://github.com/tullamods/OmniCC/issues/162#issuecomment-408558543
With Ion becoming  Neuron i thought maybe new code would fix it but its still being worked on by Omnicc author.

**Comment by spamwax** [comment]
> yeah, you can disable that countdown text right from in the bar editor window. In the bottom right box there are options for cooldown text and such

That's what I mentioned in my post that I fiddled with and got no results. It has no effect!

As I mentioned I also have to re-hide some bars every time I log in! So I think there may be another underlying issue.

**Comment by raysmith59** [comment]
This has been an issue at least as far back as Macaroon (I wasn't around in the Trinity days), unfortunately, so it's probably a legacy code issue.  I'd love to see this fixed, too, OmniCC's only real use for me is to show cooldowns on things in my bags (hearthstone, etc) but it doesn't have an option to ONLY show them in bags.  In testing a year or so ago disabling the button timers in Neuron (or Ion) also disabled them in OmniCC regardless of how I had the default UI setting set in Interface -> ActionBars -> Show Numbers For Cooldowns.  I'm assuming that's still the case.  I'm not sure if this is necessarily a Neuron issue but it seems to me that allowing an alpha of 0 for the timers would "fix" this by functionally hiding the Neuron number while still prompting OmniCC to display its own.  However, I've never been able to get that alpha setting to work.

**Comment by 40P3** [comment]
> no, the actual text that count downs to zero (right icon on the attached screenshot). I have to use omnicc because of its integration with other mods.
> 
> To me it seems to be related to saving/persisting the settings between sessions or reloads. I have to make bag bar & cast bar hidden every time i log in, so maybe hiding the timer text also doesn't get saved and I get the duplicate.[ ![world of warcraft 7_30_2018 8_06_20 pm 2](https://user-images.githubusercontent.com/1251233/43429943-637b9150-9434-11e8-91df-6b078090b47f.png) ](url)

I have this since start. Duplicate cooldown text with omni cc enabled. However in Neuron Cooldown text turned off. I go into settings, turn it off and SOMETIMES it works FOR A WHILE!
I use macros with modifiers, and I think that also add to the problem. Somehow when I push the modifier and the spell on the mod:shit e.g. is on cooldown then I start to see duplicates again.

OmniCC owner says the issue isn't on his end. :S
